src/charactersheet/viewmodels/character/saving_throws/index.js

```diff
@@ -43,10 +43,14 @@ export function SavingThrowsViewModel() {
         self.updateValues();
     };
 
-    self.updateValues = () => {
-        self.savingThrows().forEach(function(e, i, _) {
-            e.updateModifierLabel();
-        });
+    self.updateValues = async () => {
+        // By telling each savingThrow to update their labels, we're implicitly
```

Comment:
Wouldn't this not be a problem with the new caching?

Reply:
No. This is exactly where the ability scores get updated. At this point, we've just updated the score value and done our save (POST) so the cache is cleared. We then want to refetch the scores (which is what this method does, albeit entirely through side-effects). This method dispatches the API calls when KO recomputes the labels. I was just documenting that fact since this method confused me but broke things when I tried to refactor it.

This is a weird issue.